portfolio/src/main/webapp/scripts/onload.js

```diff
@@ -17,7 +17,14 @@
  */
 
 window.addEventListener('load', () => {
+  const commentSection = document.getElementById('comments_container').style;
```

Comment:
Nit: These variable names are a little confusing because you are getting the element's style and not the element itself. It may be more clearer to do something like below (and do the same for loginRequirementsMessage):

const commentSection = document.getElementById('comments_container');

commentsection.style.display = 'none';